pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1659,27 +1668,49 @@ BEGIN
     IF older_than IS NULL THEN
         -- even though there are no new Ids in need of deletion,
         -- we may still have old ones to delete
+        lastT := clock_timestamp();
+        PERFORM set_config('application_name', format('promscale maintenance: data retention: metric %s: delete expired series', metric_name), false);
         PERFORM SCHEMA_CATALOG.delete_expired_series(metric_table, ran_at, present_epoch, last_updated);
+        IF log_verbose THEN
+            RAISE LOG 'promscale maintenance: data retention: metric %: done deleting expired series as only action in %', metric_name, clock_timestamp-lastT;
+            RAISE LOG 'promscale maintenance: data retention: metric %: finished in %', metric_name, clock_timestamp()-startT;
```

Comment:
Again, when `log_verbose` is true, we should log time towards the start of the line, so that absolute time is noted, along with the time difference.

Reply:
Also, we can change the log to warn if the time taken is beyond, say 1 hour, which indicates the user should increase the background workers or more jobs for maintenance. This explaination should be mentioned in the logs as well.

Reply:
the warnings are a good idea. Added a commit.